programs/bpf_loader/src/lib.rs

```diff
@@ -1384,11 +1384,11 @@ fn execute<'a, 'b: 'a>(
         })
         .collect::<Vec<_>>();
 
+    invoke_context.execute_time = Measure::start("execute");
```

Comment:
It is not clear to me why this moved to before the `create_vm!()` call, and then after the execution the create_vm timings are subtracted from `execute_us`. Surely the `.saturating_sub(create_vm_time.as_us());` is unnecessary if the execute timer is started  just before `execute_program()`?

Reply:
`invoke_context` is borrowed by `vm`, so after `vm` is construsted we can not modify `invoke_context` any longer. We could put execute_time in a `RefCell` to avoid that.